src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/runtime:6.0 AS base
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src
+COPY ["nuget.config", "."]
+COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/"]
+COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.Common/Microsoft.Health.Fhir.Ingest.Console.Common.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.Common/"]
+COPY ["src/lib", "src/lib"]
+RUN dotnet restore "src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj"
+COPY . .
+WORKDIR "/src/src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation"
+RUN dotnet build "Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj" -c Release -o /app/publish /p:UseAppHost=false
+ 
+FROM base AS final
+WORKDIR /app
```

Comment:
We currently have two build stages in Workspace platform: `build` and `runtime`. Is there a benefit to using 3 stages here (build, publish, final)?

For 2 build stages, I was thinking of something like the following:
```dockerfile
FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
WORKDIR /src
COPY ["nuget.config", "."]
COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.FhirTransformation/"]
COPY ["src/console/Microsoft.Health.Fhir.Ingest.Console.Common/Microsoft.Health.Fhir.Ingest.Console.Common.csproj", "src/console/Microsoft.Health.Fhir.Ingest.Console.Common/"]
COPY ["src/lib", "src/lib"]
RUN dotnet publish "Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.csproj" -c Release -o /app/publish /p:UseAppHost=false

FROM base AS final
COPY --from=build /app/publish .
ENTRYPOINT ["dotnet", "Microsoft.Health.Fhir.Ingest.Console.FhirTransformation.dll"]
```

Reply:
The build step ended up not being necessary and was removed.